packages/@lightningjs/ui-components/src/mixins/withThemeStyles/index.js

```diff
@@ -456,6 +460,35 @@ export default function withThemeStyles(Base, mixinStyle) {
       this.queueThemeUpdate();
     }
 
+    replaceAliasValues(value) {
+      const styleObj = clone(value, {});
+      const aliasProps = [
+        { prev: 'height', curr: 'h', skipWarn: true },
+        { prev: 'width', curr: 'w', skipWarn: true },
+        ...(this.constructor.aliasStyles || [])
+      ];
+      aliasProps.forEach(alias => {
+        if (
+          alias &&
+          typeof alias.prev === 'string' &&
+          typeof alias.curr === 'string' &&
+          styleObj[alias.prev]
+        ) {
+          !alias.skipWarn &&
+            console.warn(
+              `The style property "${alias.prev}" is deprecated. Please use "${alias.curr}" instead.`
```

Comment:
should we add something like "and will be removed in a future version"? idk if that's implied or if being more explicit is better

Reply:
Good idea. I'd rather be as clear as possible so people can't come back and say "i DiDn'T kNoW dEpRecATing mEAnT bAAd" 🙃

Reply:
Updated to add that "____ is deprecated and may be removed in a future release." Let me know if you think that's OK or not aggressive enough.